lib/statsd/instrument/datagram.rb

```diff
@@ -64,6 +64,10 @@ def eql?(other)
 
       alias_method :==, :eql?
 
+      def key
```

Comment:
A metric is aggregated by its name + its tags: https://github.com/coralogix/statsd-coralogix-backend/blob/master/lib/coralogix.js#L159